contracts/svm/programs/express_relay/src/utils.rs

```diff
@@ -49,6 +54,50 @@ pub fn transfer_lamports_cpi<'info>(
     Ok(())
 }
 
+pub fn transfer_spl<'info>(
+    from_ta: &AccountInfo<'info>,
+    to_ta: &AccountInfo<'info>,
+    token_program: &AccountInfo<'info>,
+    authority: &AccountInfo<'info>,
+    mint: &InterfaceAccount<'info, Mint>,
+    amount: u64,
+) -> Result<()> {
+    let cpi_accounts = SplTransfer {
+        from:      from_ta.clone(),
+        to:        to_ta.clone(),
+        mint:      mint.to_account_info(),
+        authority: authority.clone(),
+    };
+
+    token_interface::transfer_checked(
+        CpiContext::new(token_program.clone(), cpi_accounts),
+        amount,
+        mint.decimals,
+    )?;
+    Ok(())
+}
+
+pub fn compute_and_transfer_fee<'info>(
+    from_ta: &AccountInfo<'info>,
+    to_ta: &AccountInfo<'info>,
+    token_program: &AccountInfo<'info>,
+    auth: &AccountInfo<'info>,
+    mint: &InterfaceAccount<'info, Mint>,
+    amount: u64,
+    fee_ppm: u64,
+) -> Result<u64> {
+    let fee = amount
+        .checked_mul(fee_ppm)
+        .ok_or(ProgramError::ArithmeticOverflow)?
+        / 1_000_000;
```

Comment:
bit of nitpick but this can overflow. For example if amount is 100M pyth tokens

Reply:
we could use bps instead of parts per million. probably sufficient precision. wdyt?

Reply:
this issue is pretty minor and can also be solved by doing arithmetic in u128 so if you feel like ppm is useful let's keep it like this and we can update the arithmetic later